src/Pages/StartPage/StartPage.js

```diff
@@ -59,8 +59,10 @@ export default function StartPage() {
   }
 }
 
+const StyledHeader =styled.header`
+max-height: 55px;
+`
```

Comment:
Looks like code formatter is not working correctly. Might be a hint for another issue.